chef/cookbooks/provisioner/templates/default/autoyast-upgrade.xml.erb

```diff
@@ -64,25 +64,28 @@
           key_re='crowbar\.install\.key=([^ ]+)'
           if [[ $(cat /proc/cmdline) =~ $key_re ]]; then
               export CROWBAR_KEY="${BASH_REMATCH[1]}"
-              echo "$CROWBAR_KEY" >/mnt/etc/crowbar.install.key
-          elif [[ -f /mnt/etc/crowbar.install.key ]]; then
-              export CROWBAR_KEY="$(cat /mnt/etc/crowbar.install.key)"
+              echo "$CROWBAR_KEY" >/etc/crowbar.install.key
```

Comment:
Do we actually still need to create that file?

Reply:
Good point, I was also wondering. But in the end I did not want to interfere in it

Reply:
Agreed, this doesn't need to be handled here.